project_template/webpack.config.js

```diff
@@ -22,6 +22,7 @@ const plugins = [
   ...require("maji/lib/webpack").plugins,
   new HTML({
     template: "src/index.html",
+    useCordova: env !== "development",
```

Comment:
I like the `userCordova`, but I wouldn't base it of development. In the CLI we know if we are building for Cordova or not, so it would be better to actually use that knowledge.

Reply:
I also had a discussion with @leonderijke about this, I want to change this to an extra ENV var, defaulting in 'useCordova' `true`. And only have the `yarn start` and `maji start` to set this to false.